### PR TITLE
Feature package ga docs

### PR DESCRIPTION
**Pull request by a-cool-train**
*Description of changes:*

*Testing (if applicable):*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/content/en/docs/concepts/packages/_index.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ description: >
 ---
 
 {{% alert title="Note" color="primary" %}}
-*The Amazon EKS Anywhere Curated Packages are only available to customers with the Amazon EKS Anywhere Enterprise Subscription. To request a free trial, talk to your Amazon representative or connect with one [here](https://aws.amazon.com/contact-us/sales-support-eks/).*
```
Comment (chrisnegus) [inline docs/content/en/docs/concepts/packages/_index.md:11]: This change is fine, but can you rebase? I made a change to this file in https://github.com/aws/eks-anywhere/pull/3024.

**Review by chrisnegus — CHANGES_REQUESTED** [review]

File: docs/content/en/docs/concepts/packages/_index.md (hunk 1 above)
Comment (a-cool-train) [reply docs/content/en/docs/concepts/packages/_index.md:11]: Done.

**Pull request merged**